### PR TITLE
Add Manibus VIA support

### PR DESCRIPTION
**Pull request by SamuraiKek**
## Description

- Added VIA support
- ~~Fixed an issue in the default keymap where the LOWER key wasn't doing anything when pressed.~~ 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/blank_tehnologii/manibus/config.h [hunk 1]
```diff
@@ -20,10 +20,10 @@
...
 
 /* define pins */
-#define MATRIX_ROW_PINS { F6, F5, F4, B6, D3 }
-#define MATRIX_COL_PINS { B5, B4, D7, D6, F0, F1, C6 }
-#define MATRIX_ROW_PINS_RIGHT { D6, D7, B4, F7, E6 }
-#define MATRIX_COL_PINS_RIGHT { B1, B2, B3, F1, F4, F5, F6 }
+#define MATRIX_ROW_PINS { F4, F5, F6, D3, C6 }
+#define MATRIX_COL_PINS { D7, B4, B5, B6, F0, D4, D6 }
+#define MATRIX_ROW_PINS_RIGHT { F5, F1, B2, B1, C6 }
+#define MATRIX_COL_PINS_RIGHT { B6, B5, D6, D5, D3, D7, B4 }
```
Comment (SamuraiKek) [inline keyboards/blank_tehnologii/manibus/config.h:26]: I don't understand why this change is still here, it already got in with #20309

File: keyboards/blank_tehnologii/manibus/keymaps/default/keymap.c [hunk 2]
```diff
@@ -57,7 +61,7 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
 //   ├────────┼────────┼────────┼────────┼────────┼────────┤                       ├────────┼────────┼────────┼────────┼────────┼────────┤
       KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,                            KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_BSLS,
 //   ├────────┼────────┼────────┼────────┼────────┼────────┼────────┐     ┌────────┼────────┼────────┼────────┼────────┼────────┼────────┤
-      KC_LCTL, KC_HOME, KC_INS,  KC_DEL,  KC_LGUI, KC_SPC,  KC_LALT,       KC_ENT,  KC_BSPC, LOWER,   KC_LBRC,  KC_RBRC,  KC_END,  KC_GRV
+      KC_LCTL, KC_HOME, KC_INS,  KC_DEL,  KC_LGUI, KC_SPC,  KC_LALT,       KC_ENT,  KC_BSPC, MO(LOWER),KC_LBRC,KC_RBRC,  KC_END,  KC_GRV
```
Comment (zvecr) [inline keyboards/blank_tehnologii/manibus/keymaps/default/keymap.c:64]: `MO(LOWER)` wont work as its a custom keycode, as such, the changes to this file should be reverted.

File: keyboards/blank_tehnologii/manibus/keymaps/via/keymap.c [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    _FN3
+};
+
+
+enum custom_keycodes {
+    QWERTY = SAFE_RANGE,
+    FN1,
+    FN2,
+    FN3
+};
```
Comment (zvecr) [inline keyboards/blank_tehnologii/manibus/keymaps/via/keymap.c:31]: Zero custom keycodes have been implemented, so this block is pointless.
```suggestion
```

File: keyboards/blank_tehnologii/manibus/keymaps/via/keymap.c [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+//   ├────────┼────────┼────────┼────────┼────────┼────────┤                       ├────────┼────────┼────────┼────────┼────────┼────────┤
+      KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,                            KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_EQL,
+//   ├────────┼────────┼────────┼────────┼────────┼────────┤                       ├────────┼────────┼────────┼────────┼────────┼────────┤
+      KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,                            KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,
+//   ├────────┼────────┼────────┼────────┼────────┼────────┤                       ├────────┼────────┼────────┼────────┼────────┼────────┤
+      KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,                            KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_BSLS,
+//   ├────────┼────────┼────────┼────────┼────────┼────────┼────────┐     ┌────────┼────────┼────────┼────────┼────────┼────────┼────────┤
+      KC_LCTL, KC_HOME, KC_INS,  KC_DEL,  KC_LGUI, KC_SPC,  KC_LALT,       KC_ENT,  KC_BSPC, MO(FN1),KC_LBRC,KC_RBRC,  KC_END,  KC_GRV
```
Comment (zvecr) [inline keyboards/blank_tehnologii/manibus/keymaps/via/keymap.c:68]: Same issue as default keymap, this also wont work. It would have to be `MO(_FN1)`.

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/blank_tehnologii/manibus/keymaps/default/keymap.c (hunk 2 above)
Comment (SamuraiKek) [reply keyboards/blank_tehnologii/manibus/keymaps/default/keymap.c:64]: Yes you're right, I got confused and thought I had an issue with the default keymap but it was actually fine before.

File: keyboards/blank_tehnologii/manibus/keymaps/via/keymap.c [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+      _______, _______, _______, _______, _______, _______,                         _______, _______, _______, _______, _______, _______,
+//   ├────────┼────────┼────────┼────────┼────────┼────────┤                       ├────────┼────────┼────────┼────────┼────────┼────────┤
+      _______, _______, _______, _______, _______, _______,                         _______, _______, _______, _______, _______, _______,
+//   ├────────┼────────┼────────┼────────┼────────┼────────┤                       ├────────┼────────┼────────┼────────┼────────┼────────┤
+      _______, _______, _______, _______, _______, _______,                         _______, _______, _______, _______, _______, _______,
+//   ├────────┼────────┼────────┼────────┼────────┼────────┼────────┐     ┌────────┼────────┼────────┼────────┼────────┼────────┼────────┤
+      _______, _______, _______, _______, _______, _______, _______,       _______, _______, _______, _______, _______, _______, _______
+//   └────────┴────────┴────────┴────────┴────────┴────────┴────────┘     └────────┴────────┴────────┴────────┴────────┴────────┴────────┘
+  )
```
Comment (zvecr) [inline keyboards/blank_tehnologii/manibus/keymaps/via/keymap.c:102]: This still needs removing, otherwise it will not compile
```suggestion
  )
```

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]
